regolith/helpers/u_contacthelper.py

```diff
@@ -0,0 +1,100 @@
+"""Helper for adding a new person to the contacts collection.
+
+"""
+import datetime as dt
+from nameparser import HumanName
+import sys
+import uuid
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+
+TARGET_COLL = "contacts"
+
+def subparser(subpi):
+    subpi.add_argument("name", help="first name space last name in quotes")
+    subpi.add_argument("institution", help="person's institution.  Can be inf"
+                                            "short form such as columbiau and will "
+                                            "be retrieved from institutions collection")
+    subpi.add_argument("-a", "--aliases", nargs='+',
+                        help="All the different ways that the person may "
+                             "be referred to as.  As many as you like, in "
+                             "quotes separated by commas")
+    subpi.add_argument("-n", "--notes", nargs='+',
+                        help="notes.  As many notes as you like, each one in "
+                             "quotes and separated by a comma, such as where"
+                             "and when met, what discussed.")
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file.")
+    return subpi
+
+class ContactUpdaterHelper(DbHelperBase):
+    """Helper for adding a new person to the contacts collection.
+    """
+    # btype must be the same as helper target in helper.py
+    btype = "u_contact"
+    needed_dbs = [f'{TARGET_COLL}']
+
+    def construct_global_ctx(self):
+        """Constructs the global context"""
+        super().construct_global_ctx()
+        gtx = self.gtx
+        rc = self.rc
+        rc.coll = f"{TARGET_COLL}"
+        if not rc.database:
+            rc.database = rc.databases[0]["name"]
+        gtx[rc.coll] = sorted(
+            all_docs_from_collection(rc.client, rc.coll), key=_id_key
+        )
+        gtx["all_docs_from_collection"] = all_docs_from_collection
+        gtx["float"] = float
+        gtx["str"] = str
+        gtx["zip"] = zip
+
+    def db_updater(self):
+        rc = self.rc
+        name = HumanName(rc.name)
+        day = dt.date.today().day
```

Comment:
you don't need day, month and year here.  If you make ``now=dt.date.today()`` (I think that is the syntax) then ``now.year`` will get you ``year``

Reply:
done!